package.json

```diff
@@ -103,6 +103,7 @@
     "babel-plugin-minify-replace": "^0.5.0",
     "babel-plugin-module-resolver": "^3.1.1",
     "babel-plugin-transform-remove-console": "^6.9.4",
+    "better-sqlite3": "^7.1.0",
```

Comment:
you _could_ add it to peerDependencies. Or maybe not.

Well, it would be nice if you could update the docs-master/ folder to mention that SQLite on NodeJS is a thing + a CHANGELOG note about your addition (making sure to mention your @name, of course)

Reply:
yeah, I was thinking about `peerDependencies` too... up to you...

Reply:
in the end, I moved to `peerDependency` so we reduce the amount of download if the person won't use it.

Also, it is possible to have an `index.web.js` to proxy `LokiJS` inside of it... making it more transparent...